src/pretty.rs

```diff
@@ -60,6 +60,17 @@ where
         self.text(s)
     }
 
+    fn multiline_string(&'a self, s: &str) -> DocBuilder<'a, Self, A> {
+        let percent_count = s.split(|c| c != '%').map(|v| v.len()).max().unwrap();
+        let delimiter = "%".repeat(percent_count + 1);
+        self.hardline()
+            .append(self.intersperse(
+                s.lines().map(|d| self.text(d.to_owned())),
+                self.hardline().clone(),
```

Comment:
Why is this clone required? It looks odd

Reply:
Good catch, I don't think it's required. Seems to be a copy&paste artefact from the old code.